src/interfaces/coral_web/src/constants.ts

```diff
@@ -67,6 +70,9 @@ export const TOOL_ID_TO_DISPLAY_INFO: { [id: string]: { icon: IconName } } = {
   [TOOL_WIKIPEDIA_ID]: { icon: 'web' },
   [TOOL_SEARCH_FILE_ID]: { icon: 'search' },
 };
+export const TOOL_ID_TO_ICON: { [id: string]: StaticImport } = {
```

Comment:
there is not need to specify the import type here since its inherit

Reply:
.svg imports are inferred as "any" in nextjs